Supervision Work/src/uk/ac/cam/ap886/oopjava/supervisionProjects/Hangman/Game.java

```diff
@@ -0,0 +1,115 @@
+package uk.ac.cam.ap886.oopjava.supervisionProjects.Hangman;
+
+import java.util.ArrayList;
+
+public class Game {
+	protected 	Dictionary 	dictionary;		
+	protected 	Player 		player;		  	
+	protected 	char[] 		etalon;			
+	protected	ArrayList 	<Character> 	guessedLetters;
+	protected	char[] 		openedWord;		
+	protected	ArrayList	<Character>		wrongLetters;
+	protected 	UserInterface ui;			
+	protected 	final int 	TOTAL_GUESSES=9;
+	protected 	int 		currentGuesses;
```

Comment:
A lot of these fields could be declared final. Would ensure that they have a value, just makes it a bit safer. But it isn't really a criticism as its not a big issue, just a pedantic failsafe :P